hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/client/HoodieFlinkWriteClient.java

```diff
@@ -309,6 +309,7 @@ public void preTxn(HoodieTableMetaClient metaClient) {
       this.lastCompletedTxnAndMetadata = TransactionUtils.getLastCompletedTxnInstantAndMetadata(metaClient);
       this.pendingInflightAndRequestedInstants = TransactionUtils.getInflightAndRequestedInstants(metaClient);
     }
+    tableServiceClient.startAsyncArchiveService(this);
```

Comment:
don't have any test for this?

Reply:
We have `TestAsyncArchiveService` for UT test, but we do not have engine specific tests for async archiving.